src/Indicator.vala

```diff
@@ -197,6 +201,15 @@ public class Power.Indicator : Wingpanel.Indicator {
         string? primary_text = null;
         string? secondary_text = null;
         if (display_device != null) {
+            if (display_device.percentage == LOW_BATTERY_PERCENTAGE && !display_device.is_charging) {
```

Comment:
This should be 
```
if (display_device.percentage <= LOW_BATTERY_PERCENTAGE && !display_device.is_charging) {

```

Otherwise the percentage may not show as the percentage may not be exactly 20.00000...%

This happened during testing - the first time the level dropped below 20% the percentage showed. However on the second time it did not.

Reply:
Done !